src/ci/ci-rerun.js

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env node
+
+const { default: chalk } = require('chalk');
+const program = require('commander');
+const got = require('got');
+
+const { fatal } = require('../utils/logging');
+
+program
+  .description('Rerun CI workflows')
+  .argument('[workflow]', 'The ID of the workflow to rerun')
+  .option('-f, --from-failed', 'Rerun workflow from failed', true)
+  .option('-e, --enable-ssh', 'Rerun workflow from failed', false)
+  .action(async (workflow, options) => {
+    try {
+      const { pipeline_number } = await got(`https://circleci.com/api/v2/workflow/${workflow}`, {
+        username: process.env.CIRCLE_TOKEN,
+        password: '',
+      }).json();
+
+      const { workflow_id } = await got
+        .post(`https://circleci.com/api/v2/workflow/${workflow}/rerun`, {
+          headers: {
+            'Content-Type': 'application/json',
+            Accept: 'application/json',
+          },
+          username: process.env.CIRCLE_TOKEN,
+          password: '',
+          json: {
+            enable_ssh: options.enableSsh,
```

Comment:
`enable_ssh` is mutually exclusive with the `from_failed` option and requires you to specify which jobs to rerun exactly.  Per docs:

https://circleci.com/docs/api/v2/#operation/rerunWorkflow